lib/savon/options.rb

```diff
@@ -10,6 +9,11 @@ def initialize(options = {})
       assign options
     end
 
+    def deprecate
+      option = caller_locations[0].label
```

Comment:
just take an argument for the option, it's better to be more explicit here, and it'll solve the behavior change that occurs in ruby-head